test_scripts/send_fwd.py

```diff
@@ -0,0 +1,47 @@
+from lsst.ctrl.iip.SimplePublisher import SimplePublisher
+from lsst.ctrl.iip.Credentials import Credentials
+import time
+
+def main():
+    cred = Credentials('iip_cred.yaml')
+    user = cred.getUser('service_user')
+    passwd = cred.getPasswd('service_passwd')
+
+    url = f'amqp://%s:%s@141.142.238.15:5672/%%2Ftest_at' % (user, passwd)
```

Comment:
We shouldn't leave the IP address in here;  consider loading this from the command line or from a configuration file.